Makefile

```diff
@@ -1,14 +1,21 @@
 guard-%:
 	@ if [ "${${*}}" = "" ]; then echo "Environment variable $* not set" && exit 1; fi
 
+define abigen
+	echo "Generating bindings for $(1)"
+    jq -r '.bytecode.object' out/$(1).sol/$(1).json > out/$(1).sol/$(1).bin
+	jq -r '.abi' out/$(1).sol/$(1).json > out/$(1).sol/$(1).abi
+	abigen --abi out/$(1).sol/$(1).abi --bin out/$(1).sol/$(1).bin --pkg bindings --type $(1) --out bindings/$(2).go
+endef
```

Comment:
passing in the `--bin` argument also adds deployment helpers to the generated bindings, which is helpful for testing